web_external/InfoView/style.styl

```diff
@@ -0,0 +1,32 @@
+.v-infoview
+  height 100%
+
+  .panel
+    height 100%
+    display flex
+    flex-direction column
+
+  .panel-body
+    flex 1
+    position relative
+
+  ul
+    &.geometry
+      padding-left 20px
```

Comment:
This is wrong. For the purpose this padding is being added, it should be in `em` or possibly in `ex` (probably `1em` to `1.5em`... I would be inclined to the low end of that).